domains/lumi.json

```
@@ -5,6 +5,7 @@
     "discord": "563697359423406082"
   },
   "record": {
+    "A": [76.76.21.21],
```

This should be a string
```suggestion
    "A": ["76.76.21.21"],
```